tools/shared/src/main/scala/org/scalajs/core/tools/linker/LinkedMember.scala

```diff
@@ -23,8 +22,7 @@ import ir.Infos
  *  reliably to determine at phase P+1 whether a linked member coming from
  *  phase P must be reprocessed.
  */
-final class LinkedMember[+T <: MemberDef](
-    val info: Infos.MethodInfo,
+final class LinkedMember[+T <: IRNode](
```

Comment:
It might make sense to rename this into `Versioned[+T]` at this point. Or else leave `LinkedMember` as is and add `LinkedTopLevelExport`.

Opinion?